superset/assets/javascripts/explore/components/ExploreActionButtons.jsx

```diff
@@ -17,9 +17,7 @@ export default function ExploreActionButtons({ canDownload, slice, query }) {
   return (
     <div className="btn-group results" role="group">
       <URLShortLinkButton slice={slice} />
-
```

Comment:
I'm just trying to be consistent with the rest of the codebase where we do not typically do this. In the absence of an enforced linting rule I'd go for consistency. I'm fine removing this from this PR.